torchbox/careers
# Add post it notes person to academy page

### Description of Changes Made

Added a post-it note person to the SEO academy page.

### How to Test

[Please see Figma designs](https://www.figma.com/file/H5pQECphbbYsvSWtag3OrCWC/Torchbox?node-id=332-1608&t=Fn66mBv4aktf0429-0).

**[Visit the preview site](https://careers-git-chore-add-new-academy-svg-careers-site.vercel.app/careers/academy)**.

### Screenshots

<img width="660" alt="image" src="https://user-images.githubusercontent.com/18164832/232543915-77eee234-4529-40c4-a067-b14ff2685bfe.png">

### MR Checklist
